src/Workspaces/Core/Portable/Remote/RemoteHostClient.cs

```diff
@@ -104,35 +62,57 @@ private void OnConnectionChanged(bool connected)
             ConnectionChanged?.Invoke(this, connected);
         }
 
-        private async Task<PinnedRemotableDataScope> GetPinnedScopeAsync(Solution solution, CancellationToken cancellationToken)
+        /// <summary>
+        /// NoOpClient is used if a user killed our remote host process. Basically this client never
```

Comment:
Aside, do we collect telemetry on this sort of thing?  I'd hate for us to introduce a problem that causes our OOP system to go down, and for us to just fail-over to local processing without collecting stats on how much this happens.